terraform/terraform_test.go

```diff
@@ -285,6 +285,22 @@ module.child:
     type = aws_instance
 `
 
+const testTerraformApplyModuleBoolStr = `
+aws_instance.bar:
+  ID = foo
+  foo = 1
+  type = aws_instance
+
+  Dependencies:
+    module.child
+
+module.child:
+  <no state>
+  Outputs:
+
+  leader = 1
+`
```

Comment:
Fun historical fact: All the tests used to live in `terraform_test.go`. It was migrated pre-0.1 to `context_test.go` except for some reason the strings were not, so they're in this file. Really all we need to do is copy and paste all of these to the bottom of `context_test.go`.